src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -405,7 +405,13 @@ class ApplicationManagerImpl
   void CreatePendingApplication(
       const transport_manager::ConnectionUID connection_id,
       const transport_manager::DeviceInfo& device_info,
-      connection_handler::DeviceHandle device_id);
+      connection_handler::DeviceHandle device_id) OVERRIDE;
+
+  void OnWebEngineDeviceCreated() OVERRIDE;
+
+  void CreatePendingApplication(const std::string& policy_app_id) OVERRIDE;
```

Comment:
Can this be named to something else other than CreatePendingApplication? There is already a method named as such.

Reply:
renamed in a8443e71f3d2c93c61eb22c33c778144f2b0a74c